front/src/applications/operationalStudies/views/Study.tsx

```diff
@@ -193,7 +193,7 @@ export default function Study() {
         };
         try {
           const filteredData = await postSearch(payload).unwrap();
-          let filteredScenarios = [...filteredData] as SearchScenarioResult[];
+          let filteredScenarios = [...filteredData] as SearchResultItemScenario[];
```

Comment:
You can delete the line 195
```suggestion
          let filteredScenarios = (await postSearch(payload).unwrap()) as SearchResultItemScenario[];
```